### PR TITLE
Follow up on `prefect profile populate-defaults`

### PR DESCRIPTION
**Pull request by zzstoatzz**
following up on some feedback that came in after https://github.com/PrefectHQ/prefect/pull/14749 was merged

this PR:
- removes `default` as the "default" profile, in favor of `ephemeral`
- shows changes to be made to the user before confirming the write to `PREFECT_PROFILES_PATH`
- updates some missed `orion` -> `server` variable names
- updates tests according to changes

image

---

### on leaving breadcrumbs within the `profiles.toml` file
- its prohibitively complex to intersperse toml with comments in a way that can be preserved between parses. there's a library [here](https://github.com/python-poetry/tomlkit) for that, but not sure its worth the squeeze

- [unfortunately](https://github.com/toml-lang/toml/issues/30#issuecomment-14003959) there is no concept of a `None / nil` in toml, and I'm not a huge fan of setting dummy "english" as values of real settings (e.g. `PREFECT_API_KEY = "run prefect cloud login to get started!"`) since we load and sporadically depend on these values existing across the codebase

I will think about ways to add breadcrumbs in a subsequent PR in a way that doesnt break things

### CODE REVIEW

**Comment by desertaxle** [comment]
Very nice! Since it seems like comments aren't preserved when updating TOML files, do you think we could include empty/placeholder values for `PREFECT_API_URL` and `PREFECT_API_KEY` to the `cloud` profile? That way, we can provide some breadcrumbs for getting started with Prefect Cloud.

File: src/prefect/cli/profile.py [hunk 1]
```diff
@@ -191,7 +192,7 @@ def delete(name: str):
     profiles.remove_profile(name)
 
     verb = "Removed"
-    if name == "default":
+    if name == "ephemeral":
```
Comment (cicdw) [inline src/prefect/cli/profile.py:195]: what's going on with this change? I'd like to avoid any special logic based on specific names

**Review by cicdw — COMMENTED** [review]
question about the special-casing of the name "ephemeral"

File: src/prefect/cli/profile.py (hunk 1 above)
Comment (zzstoatzz) [reply src/prefect/cli/profile.py:195]: `default` was previously the "default" profile that we had special logic for in a couple places. I should be able to totally remove this if we don't want any special treatment for any profiles

**Review by cicdw — APPROVED** [review]

**Pull request merged**